orttraining/orttraining/test/gradient/gradient_op_test_utils.cc

```diff
@@ -121,6 +121,51 @@ void GradientOpTester::Run(
     };
     bool has_run = false;
 
+
+    if (execution_providers) {
+      for (auto& entry : *execution_providers) {
+        if (entry->Type() == kDmlExecutionProvider) {
+          so.enable_mem_pattern = false;
+          so.execution_mode = ExecutionMode::ORT_SEQUENTIAL;
+          break;
+        }
+      }
+
+      onnxruntime::training::TrainingSession session_object{so, GetEnvironment()};
+
+      ASSERT_TRUE(!execution_providers->empty())
+          << "Empty execution providers vector.";
+      std::string provider_types;
+
+      for (auto& entry : *execution_providers) {
```

Comment:
Adding a comment here to make sure we keep this similar to whatever ends up in `OpTester::Run()`.